cypress-custom/integration/swap.test.ts

```diff
@@ -5,10 +5,9 @@
 const CHAIN_ID = 5
 const USDC = '0xD87Ba7A50B2E7E660f678A895E4B72E7CB4CCd9C'
 const WETH = '0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6'
-const QUOTE_APPLY_TIMEOUT = 2000
 
 function acceptFeesExceedWarning() {
-  cy.wait(QUOTE_APPLY_TIMEOUT)
+  cy.get('#swap-button > button').should('contain.text', 'Swap')
```

Comment:
instead wait until the price is back (the button text changes to  "Swap"), but remains disabled until you accept